tempesta_fw/http.h

```diff
@@ -177,6 +178,7 @@ typedef enum {
 
 	TFW_HTTP_HDR_CONNECTION = TFW_HTTP_HDR_NONSINGULAR,
 	TFW_HTTP_HDR_X_FORWARDED_FOR,
+	TFW_HTTP_HDR_AUTHORIZATION,
```

Comment:
There is no sense to make the header special - we don't process it's content at all.